### ISSUE TITLE
Dont get the net.minecraft Folder

### ISSUE THREAD

**Issue by LimeCodeded** [Sol-Client/Client#43]
Hello... I want to add some mods into the Client, and decompiled it. But when I decompile it, i only get the game/src Folder, and not the net.minecraft Folder, where MCP Classes are.....Where do i find this Code? Because Events may get called in Minecraft.java, and i dont get this decompiled

**Comment by LimeCodeded** [comment]
So basically i need the complete Source Code, including MCP

**Comment by TheKodeToad** [comment]
I can understand what you are trying to do, but this does not use Mod Coder Pack or MCP. Instead, it uses some of Forge's tools.

See the [build instructions](https://github.com/Sol-Client/Client#build-instructions).

**Comment by LimeCodeded** [comment]
> I can understand what you are trying to do, but this does not use Mod Coder Pack or MCP. Instead, it uses some of Forge's tools.
> 
> See the [build instructions](https://github.com/Sol-Client/Client#build-instructions).

Yeah...I run the commands of the Build instructions, but i don´t get the net.minecraft Package anyways...

**Comment by LimeCodeded** [comment]
Or do I have to install something to get these Forge Tools? I am sorry, but i never worked with them, i only worked with mcp...

**Comment by TheKodeToad** [comment]
You can still access it from the code, Sol Client just uses mixins to modify the classes instead of direct code editing.

**Comment by LimeCodeded** [comment]
Yeah okay, thanks alot

**Comment by TheKodeToad** [comment]
It already has the zoom mod. Weather changer isn't added yet.

**Comment by LimeCodeded** [comment]
Okay, i didn´t know that it has the zoom mod, than this is not working for me.....Maybe a Bug? Or its just my fault, and i am doing something wrong

**Comment by TheKodeToad** [comment]
> 

Make sure it's enabled in the mods menu. The key by default is "C". You can change it in the mod's settings.
